generator/views/lookml_utils.py

```diff
@@ -117,6 +117,16 @@ def _generate_dimensions(client: bigquery.Client, table: str) -> List[Dict[str,
     """
     dimensions = {}
     for dimension in _generate_dimensions_helper(client.get_table(table).schema):
+        # Rename `event` field in `events_stream` to `full_event_name` to expose it in Looker views.
+        # This is required since `event_timestamp` is renamed to `event` in `_get_dimension` and takes precedence below.
```

Comment:
IMO it would be better to change the other logic to not have dimension groups conflict in this way, since they wouldn't actually conflict in the generated LookML, as dimension groups generate dimensions with timeframe suffixes.

Reply:
https://github.com/mozilla/lookml-generator/pull/1010